### ISSUE TITLE
Event user_id not included in Event struct

### ISSUE THREAD

**Issue by andrewjmead** [stripe/stripe-go#80]
I'm working on a 3rd party stripe app. I saw in [the webhook docs](https://stripe.com/docs/connect/getting-started#webhooks) that a `user_id` property will be returned if your using Stripe Connect so you can identify the stripe user the even belongs to.

While developing I was not getting the user_id in the Event instance. I noticed that the Event struct does not include that UserId property [event.go#L7](https://github.com/stripe/stripe-go/blob/master/event.go#L7), so that JSON field is never decoded.

I made a quick change to `event.go` to see if I could fix it, and everything was working as expected:

```
type Event struct {
	ID       string     `json:"id"`
	Live     bool       `json:"livemode"`
	Created  int64      `json:"created"`
	Data     *EventData `json:"data"`
	Webhooks uint64     `json:"pending_webhooks"`
	Type     string     `json:"type"`
	Req      string     `json:"request"`
	UserId   string     `json:"user_id"` // The new line
}
```

If I created a PR to fix this, would it get merged?

**Comment by cosn** [comment]
Sure, that's be great!

**Comment by cosn** [comment]
Fixed in 019138ae29961efd68dfc80427007c150938664c. Thanks!

**Comment by andrewjmead** [comment]
Thanks @cosn!
